openlibrary/templates/covers/add.html

```diff
@@ -6,15 +6,19 @@
 $if doc.type.key == "/type/author":
     $ intro = _("There are two ways to place an author's image on Open Library")
     $ action = doc.url('/add-photo')
+    $ guideline = _("Image Guidelines")
 $elif doc.type.key == "/type/edition":
     $ intro = _("There are two ways to get a cover image on Open Library")
     $ action = doc.url('/add-cover')
+    $ guideline = _("Cover Guidelines")
 $elif doc.get_edition_covers:
     $ intro = _("There are three ways to get a cover image on Open Library")
```

Comment:
Question for reviewers...  Should line 15 say `three ways` or `two ways` like the other instances?